aclk/agent_cloud_link.c

```diff
@@ -5,20 +5,30 @@
 
 // Read from the config file -- new section [agent_cloud_link]
 // Defaults are supplied
-int aclk_recv_maximum = 0; // default 20
-int aclk_send_maximum = 0; // default 20
 
 int aclk_port = 0;          // default 1883
```

Comment:
We should change this to `#DEFINE` and remove the comment `// default 1883` because then the comments are not maintained (we are already using 9002 as default because of wss).
The same goes for aclk_hostname.

Reply:
Good idea. I will add define for this `ACLK_DEFAULT_PORT` and also add `ACLK_DEFAULT_HOST`